From db23b30525822262cab7ca2219ecd23690cdc320 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 27 Feb 2015 16:05:06 -0500
Subject: [PATCH 171/315] manager: reset session-type for autologin

If we're doing an autologin we don't want to inadvertently do
wayland for X11 sessions, so make sure to reset it.
---
 daemon/gdm-manager.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/daemon/gdm-manager.c b/daemon/gdm-manager.c
index cfcfa03f..1728b357 100644
--- a/daemon/gdm-manager.c
+++ b/daemon/gdm-manager.c
@@ -1505,6 +1505,7 @@ set_up_greeter_session (GdmManager *manager,
                  gdm_display_set_up_greeter_session (display, &allowed_user);
         } else {
                  g_object_set (G_OBJECT (display), "session-class", "user", NULL);
+                 g_object_set (G_OBJECT (display), "session-type", NULL, NULL);
                  allowed_user = g_strdup ("root");
         }
 
-- 
2.15.1

